linebacker/src/main/scala/io/chrisdavenport/linebacker/Linebacker.scala

```diff
@@ -1,36 +1,44 @@
 package io.chrisdavenport.linebacker
 
-import cats._
-import cats.implicits._
-import cats.effect.Async
+import cats.effect._
 import java.util.concurrent.ExecutorService
 import scala.concurrent.ExecutionContext
 
 trait Linebacker[F[_]] {
 
-  def blockingPool: ExecutionContext
+  def blockingContext: ExecutionContext
+
+  @deprecated("0.2.0", "Use blockingContext instead.")
+  private[linebacker] def blockingPool: ExecutionContext = blockingContext
+
+  @deprecated("0.2.0", "Use blockShift instead.")
+  final private[linebacker] def block[A](
```

Comment:
just because I got trolled, what's the point in only deprecating and just just outright removing when you don't _need_ bincompat  for 0.2 yet?